project.clj

```diff
@@ -95,7 +95,7 @@
              :ezbake {:dependencies ^:replace [[puppetlabs/puppet-server ~ps-version]
                                                [puppetlabs/trapperkeeper-webserver-jetty9 ~tk-jetty-version]
                                                [org.clojure/tools.nrepl "0.2.3"]]
-                      :plugins [[puppetlabs/lein-ezbake "0.2.3"]]
+                      :plugins [[puppetlabs/lein-ezbake "0.3.0-SNAPSHOT"]]
```

Comment:
No it won't work until we also publish 0.3.0-Snapshot into nexus. It's more here in this PR for early review and less to merge right this moment. 



—
Jeff

On Tue, Feb 24, 2015 at 8:32 AM, Jeremy Barlow <notifications@github.com>
wrote:

>> @@ -95,7 +95,7 @@
>>               :ezbake {:dependencies ^:replace [[puppetlabs/puppet-server ~ps-version]
>>                                                 [puppetlabs/trapperkeeper-webserver-jetty9 ~tk-jetty-version]
>>                                                 [org.clojure/tools.nrepl "0.2.3"]]
>> -                      :plugins [[puppetlabs/lein-ezbake "0.2.3"]]
>> +                      :plugins [[puppetlabs/lein-ezbake "0.3.0-SNAPSHOT"]]
> Will running from this SNAPSHOT work - at least temporarily - in CI?  On the CI node that runs ezbake will lein-ezbake have already been built from latest master and installed into a local .m2/repository so that the dependency can be satisfied?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/puppetlabs/puppet-server/pull/411/files#r25266797